documentation/app/styles/app.scss

```diff
@@ -0,0 +1,28 @@
+/**
+* Copyright (c) HashiCorp, Inc.
+*/
+
+@import "@hashicorp/consul-ui-toolkit";
+@import "@hashicorp/design-system-components";
+
+/**
+* the styles to be removed once documentation setup is merged
+ */
+pre, code {
+  border-radius: 6px;
+  background-color: black;
+  color: white;
+  font-size: 14px;
+  width: fit-content;
+  padding-right: 20px;
+}
+
+.list-container {
```

Comment:
suggestion: always namespace the class names (eg. `doc-`) or pretty soon you may risk to end up with conflicts in CSS class names